management/src/main/java/com/bitwisekaizen/sdss/dto/IscsiTarget.java

```diff
@@ -0,0 +1,49 @@
+package com.bitwisekaizen.sdss.dto;
+
+/**
+ * ISCSI target currently managed by the SDSS.
+ */
+public class IscsiTarget {
```

Comment:
This isn't really an iscsi target ... the iscsitargetspec is.  This is more of a entity that describes the storage service agent and all of the iscsi targets on it.  I'd call this LioBackedAgent or something.